apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md

```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
+Second question: No, there is a different editor in the Personalize product.
 
-**Does Personalize work at the component level to swap out content or just pop-ups? **
+### Does Personalize work at the component level to swap out content or just pop-ups?
 
 Personalization is not restricted to just pop-up functionality but instead can apply to page level or component level variations. Offers stored in Sitecore Personalize paired with Web Experiences can allow marketers to customize various parts of a webpage, from within Sitecore Personalize using Html/CSS/JavaScript. In addition, using a Full Stack headless experience, any content irrespective of the source (including XM or Content Hub) can be personalized.
 
-**Will my potential Personalize implementation still be compatible and work if I later move into XM SaaS?**
+### Will my potential Personalize implementation still be compatible and work if I later move into XM Cloud?
 
 If the user has purchased Standalone Personalize (or CDP, SmartHub CDP) then they will keep that instance if they move to XM Cloud. The embedded Personalization in Pages will be enhanced with additional condition templates and the user will be able to update the experiences created in Pages by going into the Sitecore Personalize app.
 
-**Do I need to make changes to my current XM/XP implementation in order to utilize Sitecore Personalize (SP)?**
+## Do I need to make changes to my current XM/XP implementation in order to utilize Sitecore Personalize (SP)?
 
 Technically, you could use XP and Personalize side by side and could even be some benefits of doing so for a temporary period of time to collect data before fully embracing Personalize. Technically there would be some changes necessary, however the impacts would be minimal. Please refer to developer portal integration documents.
 
 With an existing XP or XM on-prem implementation, to start utilizing Sitecore Personalize, you will need to integrate the web tracking mechanisms into your existing XP/XM instance. This includes placing the Sitecore CDP/Personalize JS Library (Boxever.js) and then adding tracking events in JavaScript, to track view and identity events where appropriate. There are details about these required changes in the Developer Portal ([https://developers.sitecore.com/learn/integrations/xm-smarthub-cdp](https://developers.sitecore.com/learn/integrations/xm-smarthub-cdp))
 
-**I see that Sitecore Personalize (SP) has content (offers), can you store these in Content Hub or XM? Is there a single source of content?**
+## I see that Sitecore Personalize (SP) has content (offers), can you store these in Content Hub or XM? Is there a single source of content?
 
 No, you currently cannot store offers in Content Hub or XM and pull these into Sitecore Personalize. However, you can pull content from XM or Content Hub when personalizing a piece of content using a Full Stack experience vs a Web Experience using custom code in your front end.

```

Comment:
I would maybe rephrase this whole section.. Here's what I have using Chat GPT:

There are two approaches to handling a single content source. The first approach involves using a Decision Model with a Web Experience, which directly pulls content from your content management system within the Decision Model, and then displays the result in your Web Experience. The second approach is to use a Full Stack Interactive experience, where your frontend code requests a decision, receives a JSON result, and then retrieves the correct content items.

Reply:
I have updated the answer to **Is there a single source of content?** with the text you supplied